scrapy/crawler.py

```diff
@@ -152,6 +153,8 @@ def start(self, stop_after_crawl=True):
         if self.settings.getbool('DNSCACHE_ENABLED'):
             reactor.installResolver(CachingThreadedResolver(reactor))
 
+        tp = reactor.getThreadPool()
+        tp.adjustPoolsize(maxthreads=self.settings.getint('REACTOR_THREADPOOL_MAXSIZE'))
```

Comment:
The default is "0". 
https://github.com/twisted/twisted/blob/trunk/twisted/internet/base.py#L955
It hits assert if **minthreads is greater than maxthreads**. 
I think, we're fine here.